docs/guides/README.md

```diff
@@ -40,11 +40,17 @@ System APIs of Thunder are secured by default. You need to obtain an access toke
 
 This documentation is organized into the following sections:
 
+### [Standards-Based Flows](standards-based/)
```

Comment:
Shall we use a different term for "Flows"? Someone could get confused with orchestrated flows.

Need to update internal documents as well. I.e. docs/guides/standards-based, etc